numpy/distutils/misc_util.py

```diff
@@ -14,9 +14,15 @@
 import distutils
 from distutils.errors import DistutilsError
 try:
-    from threading import local as tlocal
+    from threading import local as tlocal, Lock
 except ImportError:
-    from dummy_threading import local as tlocal
+    from dummy_threading import local as tlocal, Lock
+
```

Comment:
Let me see if I understand. The `_tdata` instance holds thread local data but is created when the module is executed, probably in the main thread? What if the module is imported in different threads, is it only executed once?